models/src/test/java/io/sphere/sdk/reviews/ReviewImplTest.java

```diff
@@ -0,0 +1,71 @@
+package io.sphere.sdk.reviews;
+
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by njacinto on 21/04/15.
+ */
+public class ReviewImplTest {
```

Comment:
I would skip this test, since we should test in a ReviewCreateCommand if all getters work as expected and match the draft object.